selfdrive/test/process_replay/update_model.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+from selfdrive.test.openpilotci_upload import upload_file
+from selfdrive.test.process_replay.compare_logs import save_log
+from selfdrive.test.process_replay.test_processes import segments, get_segment
+from selfdrive.version import get_git_commit
+from tools.lib.logreader import LogReader
+from inject_model import inject_model
+
+if __name__ == "__main__":
```

Comment:
This is only updating the routes in process_replay. We also need to do the ones in test_car_models.py